index.html

```diff
@@ -1,19 +1,127 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <!-- Body Starts Here -->
+  <body>
+    <!-- Header Starts Here -->
+    <header class="header">
+      <img
+        src="img/karma-logo.svg"
+        width="100%"
+        alt="karma-logo"
+        class="navbar-logo"
+      />
+      <nav>
+        <button>
+          <a href="#"><strong>Meet Karma</strong></a>
+        </button>
+        <button><a href="#">How it Works</a></button>
+        <button><a href="index.html">Store</a></button>
+        <button><a href="#">Blog</a></button>
+        <button><a href="#">Help</a></button>
+        <button><a href="#">Login</a></button>
+      </nav>
+    </header>
 
-</body>
+    
+    <!-- Main Starts Here -->
+    <main class="main">
+      <!-- Article One Starts Here -->
+      <div class="first-section">
+        <h1 style="font-size: 4vw">Introducing Karma</h1>
+        <h3 style="font-size: 2vw">Bring WiFi with you, everywhere you go.</h3>
+        <button class="button-orange">Learn More</button>
+      </div>
+
+      <!-- Article Two Starts Here -->
+      <h1 style="font-size: 3vw">
```

Comment:
try to not mix styling on the object and css, it could make harder to find the source of a specific change if you need to